src/promptflow-devkit/promptflow/_sdk/entities/_flows/dag.py

```diff
@@ -46,8 +46,8 @@ def name(self) -> str:
         return self._flow_dir.name
 
     @property
-    def flow_dag_path(self) -> Path:
```

Comment:
this is a breaking change to an public attribute?

Reply:
It's ok. We haven't announced this class public yet. 

https://microsoft.github.io/promptflow/reference/python-library-reference/promptflow-devkit/promptflow.entities.html

Reply:
revert it.